runtime/drivers/druid/druid.go

```diff
@@ -42,7 +42,6 @@ var spec = drivers.Spec{
 			DisplayName: "Connection string",
 			Placeholder: "https://example.com/druid/v2/sql/avatica-protobuf?authentication=BASIC&avaticaUser=username&avaticaPassword=password",
 			Secret:      true,
-			NoPrompt:    true,
```

Comment:
These `dsn` should not be NoPrompt.

Reply:
I see, if that's the case. I'll revert these NoPrompt: true removal and do not filter out these NoPrompt from the UI.